doc/source/whatsnew/v1.2.0.rst

```diff
@@ -157,7 +157,7 @@ ExtensionArray
 
 Other
 ^^^^^
--
+- Bug in :meth:`DataFrame.shift` with ``axis=1`` and heterogeneous dtypes (:issue:`35488`)
```

Comment:
can you move to 1.1.1 (and put in the regression section)

Reply:
done